.github/workflows/kind.yaml

```diff
@@ -34,7 +34,7 @@ jobs:
           IMG=flotta-operator:latest make docker-build
           kind load docker-image flotta-operator:latest
           timeout 2m make deploy IMG=flotta-operator TARGET=kind || retVal=$?
-          if [ $retVal -ne 124 ]; then
+          if [ "$retVal" -ne "124" ]; then
```

Comment:
Did you try ` if [[ $retVal -ne 124 ]]; then` ?
If i remember correctly `-ne` only works with ``[[``

(Not sure :) )

Reply:
I tried that now, but with this syntax the condition is true (`$retVal != 124` is true), so I received `Make run failed` and kind exits (even though it shouldn't)

Reply:
IMO, `retVal` is unset.

if `timeout 2m make deploy IMG=flotta-operator TARGET=kind` doesn't fail, then the `retVal=$?` isn't executed (there is the `||` operator! ).
So ` $retVal -ne 124` is true and that's correct.

You can try
```
> false || echo $?
1
> true || echo $?
```

and

```
#!/bin/bash

retVal=123
if [[ $retVal -ne 124 ]]
    then
        echo "Then branch"
    else
        echo "Else branch"
fi

unset retVal
if [[ $retVal -ne 124 ]]
    then
        echo "Then branch"
    else
        echo "Else branch"
if
```

It returns:
```
> ./test.sh
Then branch
Then branch
```
In addition, `-ne` shouldn't be used with strings (see [bash conditional expression](https://www.gnu.org/software/bash/manual/html_node/Bash-Conditional-Expressions.html), [here](https://www.delftstack.com/howto/linux/bash-ne-not-equal/)  or [here](https://linuxtect.com/linux-bash-not-equal-ne-operators-tutorial/) for example)

Reply:
Nice! so I'll update the condition to check whether `retVal` is set and only if it is, it will check its value:
`if [[ -n $retVal && $retVal -ne 124 ]]; then`

Reply:
I'm sorry to bother you again on this but can you please check if `-n` works with non-string?
Since `-n` checks if a string is non-empty, I think we should use `-n "$retVal"` instead of `-n $retVal`.
(I'm still refering on [Bash conditional expression](https://www.gnu.org/software/bash/manual/html_node/Bash-Conditional-Expressions.html))